### ISSUE TITLE
Do not delete users

### ISSUE THREAD

**Issue by MoNTE48** [silverwind/droppy#300]
Hi. I can't remove user. I press Trash, but nothing happens

**Comment by silverwind** [comment]
There was a recent bug with user deletion that I fixed in https://github.com/silverwind/droppy/commit/ce83d22e34a5c0cfa08a04c30aa6a05e747e5203. Can you update to 6.3.15?

**Comment by silverwind** [comment]
Hmm, then I need more details. Can you check if there's any error in the browser console or on the server log?

**Comment by MoNTE48** [comment]
browser console? what you mean?

**Comment by silverwind** [comment]
Hit F12 to open the browser's DevTools and look for errors in the Console tab after your press the button. Thought, I still almost certain you have the exact bug I fixed earlier. Hit CTRL-Shift-R while on the site in your browser to do a no-cache reload and then try deleting again.

**Comment by MoNTE48** [comment]
oops, sorry. i use 6.3.13 (last version on npm)

**Comment by r15ch13** [comment]
Same problem here. Version 6.3.13

No console error or output.
Websocket frame for each click looks like this:
```
{"vId":null,"type":"UPDATE_USER","data":{"pass":""},"token":"fd320ef0008c5ba2738b54a627290314"}
```

**Comment by silverwind** [comment]
Ah, I was having some issues on my deployment machine that lead to me skipping the push steps on 6.3.14 and 6.3.15. It's fixed now and 6.3.16 is published which should also resolve this.
